py-polars/tests/unit/operations/test_cast.py

```diff
@@ -220,6 +220,8 @@ def _cast_lit(
         (2**15 - 1, pl.UInt16, pl.Int16, True, 2**15 - 1),
         (2**31 - 1, pl.UInt32, pl.Int32, True, 2**31 - 1),
         (2**63 - 1, pl.UInt64, pl.Int64, True, 2**63 - 1),
+        (2**63 - 1, pl.UInt64, pl.Boolean, True, True),
+        (0, pl.Int64, pl.Boolean, True, False),
```

Comment:
The tests in this file test the ChunkedArray casts, not AnyValue casts.

It's nice to have these tests, but could you open a separate new PR for them?